jb/research.md

```diff
@@ -54,157 +58,172 @@ Studies on cash transfers in sub-Saharan Africa tend to report 3 such metrics, n
-
-These small and inconsistent effects may result from the long delays between nutrition and its anthropomorphic benefits {cite}`SouthAfrica_CSG_final` {cite}`Ghana_final` {cite}`Zambia_CGP_final`.
+### Health
+While food security and nutrition are improved markedly by cash transfers, there is weaker evidence of an impact on child health outcomes. 
+Across child anthropomorphic outcomes, healthcare utilization, morbidity and mental health outcomes, there is only limited evidence that child health improves as a result of a cash transfer. 
 
+The GiveDirectly Rwanda program found small positive impacts of transfers on two anthropomorphic measures, Height-for-age and Weight-for-age (0.091 standard deviations\*\* [0.15] and 0.067 standard deviations\*\* [0.15], respectively), as did the South African CSG.
+However, most other evaluations, including the Ghana LEAP and Zambia CGP, found null effects. This is perhaps particularly surprising as the Ghana LEAP also provided free enrolment in a national health insurance program.
+The Lesotho CGP evaluation even found a significant reduction in Weight-for-age amongst program recipients, though there was no corresponding increase in the proportion of children suffering from malnutrition.
+These small and inconsistent effects may result from the long delays between nutrition and its anthropomorphic benefits.
 There is also some evidence that the gender of the recipient affects the impact of cash transfers on health anthropomorphic outcomes.
-The South African Old Age Pension only had significant impacts on Weight-for-height for girls if their grandmother (and not grandfather) was eligible for the pension {cite}`SA_Duflo` . 
+The South African Old Age Pension only had significant impacts on Weight-for-height for girls if their grandmother (and not grandfather) was eligible for the pension.
 
 Despite improving access to healthcare being a central aim of many cash transfers, there is little evidence that household expenditure on health-related services increases with receipt of a cash transfer.
-The Burkino Faso NCTPP and Malawi SCTP report significantly higher utilization of healthcare services (and particularly curative services) on receipt of the transfer {cite}`BurkinaFaso_final` {cite}`Malawi_SCTP_final`.
-However, the World Bank meta-analysis finds no significant divergence in mean expenditure on health services and both the Tanzania TASAF and Zimbabwe HSCT actually report decreased expenditure {cite}`World_Bank_meta` {cite}`Tanzania_TASAF_final` {cite}`Zimbabwe_HSCT_final`. 
+The Malawi SCTP reports significantly higher utilization of healthcare services (and particularly curative services) on receipt of the transfer.
+However, the World Bank meta-analysis finds no significant divergence in mean expenditure on health services and both the Tanzania TASAF and Zimbabwe HSCT actually report decreased expenditure. 
```

Comment:
I see HSCT is a UCT; is TASAF conditioned on healthcare activities?

Should this be surprising if cash transfers improve nutrition and other means of improving general health, requiring less healthcare spending?

In general it seems like, if the message is uncertain results from UCTs, leading with the World Bank meta-analysis and then mentioning that SCTP raises while TASAF and HSCT reduce, could be less bouncy than the current flow going into the NCTTP sentences.